### ISSUE TITLE
A11y_Visual Studio Code_Docs Setup_Usability: Keyboard focus on links is getting overlapped with link text

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4518]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_Visual Studio Code#A11ySev4#A11yTCS#A11yUsable#Chrome#DesktopWeb#Win10#Visual Studio Code Services#VisualStudioCode-Web-Apr2021

### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/docs/)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)

### Note:
This issue is observed for all links through out the application

### Repro Steps:

1. Hit [Visual Studio Code](https://code.visualstudio.com/docs/) link
2. Tab till Docs link in top navigation section and press enter
3. Tab till Set up in left navigation pane and select Linux/Mac OS
4. Tab till any link present in Linux page
5. Verify that keyboard focus gets overlapped with link text

### Actual Result:
Keyboard focus on links is getting overlapped with link text

### Expected Result:
Keyboard focus on links should not get overlapped with link text. Link text should be clearly visible when any link is focused

### User Impact:
End Users will not be able to understand the text properly if focus overlaps with text

### Attachment:

https://user-images.githubusercontent.com/83570638/117013149-39277780-ad0d-11eb-96c3-80cd6d15ab6e.mp4
